evap/staff/tests/test_views.py

```diff
@@ -529,6 +531,48 @@ def test_wrong_files_dont_crash(self):
         self.assertIn("An error happened when processing the file", reply)
 
 
+class TestUserExportView(WebTestStaffMode):
+    url = "/staff/user/export"
+
+    @classmethod
+    def setUpTestData(cls) -> None:
+        cls.manager = make_manager()
+        baker.make(UserProfile, _quantity=5, _fill_optional=["first_name_given", "last_name", "email", "title"])
+        baker.make(
+            UserProfile,
+            _quantity=5,
+            _fill_optional=["first_name_given", "last_name", "email", "title"],
+            is_active=False,
+        )
+        # the titles are not filled by baker
```

Comment:
They should be if passed as `_fill_optional`. If that doesn't work, we should probably open an issue with them. You can pass an iterator to `baker.make` as the `title` kwarg, it will then use these values.

Reply:
It had an default value, so it is not filled